index.js

```diff
@@ -201,6 +200,27 @@ function session(options) {
       return;
     }
 
+    /**
+     * Load a `Session` instance via the given `sid`
+     * and invoke the callback `fn(err, sess)`.
+     *
+     * @param {String} sid
+     * @param {Function} fn
+     * @api public
+     */
+    store.load = function(sid, fn){
```

Comment:
This would override any load method or property a store has defined. We need to validate that this will not conflict with any existing store implementations.

Reply:
So, sorry, excuse me if I'm confused is all 😂 . So if none of the stores are implementing this besides what they get from the store base, why perform this override? Why not simply modify the base method?

Reply:
I'm unsure what you mean, but it sounds like this is going to introduce some sort of race condition. I didn't realize you were adding a method to the store (effectively a global) while closing over a request... would not a call to req.session.load by a user then then up with a different internal req reference?